### ISSUE TITLE
When selecting addon, no progress showing or notification if it was successful.

### ISSUE THREAD

**Issue by vbfischer** [WilliamMayor/firestorm#19]

**Comment by WilliamMayor** [comment]
Yeah, I started on some progress notification stuff (using WebWorkers if I
remember correctly) I don't think I ever got it to a real working state.

I see a red bar at the top that starts at about 30% in and then jumps to
100% after some work. Not really a progress meter :)

---

William Mayor
